server_environment/models/server_env_tech_name_mixin.py

```diff
@@ -28,9 +29,6 @@ class ServerEnvTechNameMixin(models.AbstractModel):
 
     _name = "server.env.techname.mixin"
     _description = "Server environment technical name"
-    _sql_constraints = [
```

Comment:
I disagree w/ this change. 
The main reason being: the name must be unique per ENV not per company -> how do you make sure you set the right name for different companies in the same env?
You must make sure that were you use this mixin the tech name is computed properly (eg: by prepending the company name or a tech name for the company).

Reply:
IMO it should be explicit and you should have 2 entries in the env conf.

Regarding the normalization of the name: it's better to have a tech field on the company too because you don't want to lose the conf if someone changes the name (that's why "tech_name" is used here). You could use the ID of the company if you prefer.